# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: dc2ad0369b86
# Full Hash: dc2ad0369b8660e3a5e039e03e224122eb46dcfc
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-09 21:19:46
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 9fb8ccb2f305 -r dc2ad0369b86 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Apr 09 12:31:55 2024 +0000
+++ b/modules/libpref/init/all.js	Tue Apr 09 12:31:56 2024 +0000
@@ -1203,7 +1203,7 @@
 // NOTE: support for "compress" has been disabled per bug 196406.
 // NOTE: separate values with comma+space (", "): see bug 576033
 pref("network.http.accept-encoding", "gzip, deflate");
-pref("network.http.accept-encoding.secure", "gzip, deflate, br");
+pref("network.http.accept-encoding.secure", "gzip, deflate, br, zstd");
 
 // Prompt for redirects resulting in unsafe HTTP requests
 pref("network.http.prompt-temp-redirect", false);